mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,16 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function getMood(value) {
+  if (value === true) {
```

Comment:
probably u know about this buddy, but I have to let you know so i can tick the commenting assignment box off the list :)

you could've just left value without if statement, because as long as the value is not one of the falsies it always elevates to true (or this is how i undrerstand it).

best, 
see u tomorrow.